docs/configuration/identity-providers/oidc.md

```diff
@@ -89,91 +89,103 @@ for which stage will have each feature, and may evolve over time:
 
-### issuer_private_key
+#### [2] issuer_private_key
 
-The private key in DER base64 encoded PEM format used to encrypt the [OpenID Connect] JWT's.
+The private key in DER base64 encoded PEM format used to encrypt the [OpenID Connect] JWT's. Can also be defined using a [secret](../secrets.md) which is the recommended for containerized deployments. The reason for using only the private key here is that one is able to calculate the public key easily from the private key in this format (`openssl rsa -in rsa.key -pubout > rsa.pem`).
 
-Can also be defined using a [secret](../secrets.md) which is the recommended for containerized deployments.
+You must generate this option yourself. To create it, use `docker run -u "$(id -u):$(id -g)" -v "$(pwd)":/keys docker.io/authelia/authelia:latest authelia rsa generate --dir /keys` to generate both the private and public key in the current directory. You can then paste the private key into your configuration. When using Kubernetes, remember to base64-encode the private key first when using a secret. Using secrets is always recommended.
 
-### clients
+#### clients
 
 A list of clients to configure. The options for each client are described below.
 
-#### id
+##### [3] id
 
-The Client ID for this client. Must be configured in the application consuming this client.
+The Client ID for this client. Must be configured in the application consuming this client as well and must match with the entry in this configuration.
 
-#### description
+##### [4] description
 
 A friendly description for this client shown in the UI. This defaults to the same as the ID.
 
-#### secret
+##### [5] secret
+
+The shared secret between Authelia and the application consuming this client. Currently this is stored in plain text. This secret must be configured in the application consuming this client as well and must match with the entry in this configuration.
+
+You must generate this option yourself. To generate a random string of sufficient length, you can use `openssl rand -base64 32`. If you are deploying this as a Kubernetes secret, you must encode it with base64 again, i.e. `openssl rand -base64 32 | base64`. Using secrets is always recommended.
+
+##### [6] authorization_policy
+
+The authorization policy for this client: either `one_factor` or `two_factor`.
+
+##### [7] redirect_uris
 
-The shared secret between Authelia and the application consuming this client. Currently this is stored in plain text.
+A list of valid callback URL´s this client will redirect to. All other callbacks will be considered unsafe. The URL's are case-sensitive. This differs from application to application - we have provided a list of URL´s for common applications below. If you do not find the application in the list below, you will need to search for yourself - and maybe come back to open a PR to add your application to this list so others won't have to search for them.
```

Comment:
It's good that you listed the apps that you use. That can really help the community. However, I'd rather move this matrix under the community section for the reasons mentioned here: https://github.com/authelia/authelia/issues/2136.
Basically, the core team cannot commit to track the changes happening on all applications in the wild. However this page of the documentation is what we actually commit to support to make it 100% accurate and relevant.

Consequently, the matrix should instead be a community work located under the community section with a comment at the top stating that this work comes from the community and cannot be guaranteed to be 100% accurate. Indeed, the state of this matrix can never be guaranteed to be true unless there are some automated tests for each item running regularly or some people are committing to check each line at least regularly enough.

Do you see my point?

Reply:
Agreed.